routes/ama.js

```diff
@@ -14,6 +16,16 @@ var queries = {
     update: "UPDATE ama_questions SET answer_text = "
 };
 
+let transporter = nodemailer.createTransport({
```

Comment:
Thoughts on pulling this up into `functions.js`? Then we'd only have to create a transport once, along with requiring the email config and nodemailer in only one place.

Reply:
I can take a stab at it. Mind if I push to this branch?